mlflow/statsmodels.py

```diff
@@ -473,61 +452,36 @@ def results_to_dict(results):
 
         return results_dict
 
-    def wrapper_fit(original_method):
```

Comment:
I don't think there's a need for the patch implementation to call the `original_method` of a superclass from a subclass. As discussed in https://github.com/mlflow/mlflow/pull/3859/files#r544936250, `safe_patch` will handle parent/child patch ordering by way of `mlflow.utils.gorilla` and will deliver the correct `original` function as the first parameter to the patch implementation (`original`).

@harupy Please let me know if I'm missing anything here.

Reply:
We fixed this child-parent issue in https://github.com/mlflow/mlflow/pull/3582, right?

Reply:
Yep! We did!